src/quo2/core.cljs

```diff
@@ -323,6 +324,7 @@
 
 ;;;; Wallet
 (def account-card quo2.components.wallet.account-card.view/view)
+(def account-origin quo2.components.wallet.account-origin.view/view)
```

Comment:
This should swap lines with account-overview to be alpha sorted.

Reply:
But `or` should be before `ov`